src/pages/components/modal/usage.mdx

```diff
@@ -1,103 +1,256 @@
+
+## Anatomy
+
+<Row>
+<Column colLg={8}>
+
+![Modal anatomy diagram](images/modal-usage-anatomy.png)
+
+</Column>
+</Row>
 
 The modal is composed of three distinct zones: A header, the body, and a footer. Components (eg. data table, form, progress indicator) can occupy the full width of the modal.
 
-#### Header
+1. **Header**: Includes a title, optional label, and the close icon. The title should be brief and clearly describe the dialogs’s task or purpose. Use the optional label above the title set the context for the information in the dialog.
+
+2. **Body**: Contains the information and/or controls needed to complete the modal's task. It can include message text and components. A modal should include only content relevant to completing the current task. Text should only be 80% of the modal’s width and components can span 100% of the width.
+
+3. **Footer**: Contains the main actions needed to complete or cancel the dialog task. Button groupings change based on modal variant. Use descriptive words for the actions like Add, Delete, Save and avoid vague words like Done or OK. If you need to include a “docs” or other non-primary action, include it as a link in the modal’s body.
+
+4. **x**: The close `x` icon will close the dialog without submitting any data.
 
-The header of your modal should mirror the action that launched the modal. Headers must include a close button “✕” in the upper right-hand corner of the modal. You can also include an optional label above your header text. This is an opportunity to offer additional context.
+5. **Overlay**: Screen overlay that obscures the on page content.
+
+## Modal sizes
+
+There are four preset modal sizes: **xs, small, default, large**. They have responsive widths that change based on the screen size. Modal widths are defined as screen percentages that will complement columns on the 2x grid. See the modal spec for specifics.
+
+Choose a size best for the amount of modal content you have. Modals with short messages should use a xs or small modal to avoid long single lines; for complex components, like data table the default or large modal will be more accommodating.
+
+<Row>
+<Column colLg={8}>
 
-#### Body
+![Modal sizes](images/modal-usage-sizes.png)
 
-A modal should have minimal body content. Components that may be used in modals include: form fields, text area, select, and radio buttons. Text, including the paragraph component, should only be 75% of the modal's width.
+</Column>
+</Row>
 
-#### Footer
+#### Title as message
 
-The footer area of a modal typically contains either one or two [buttons](/components/button/code). Do not include three buttons in the footer of your modal. If you need to include a “help” or other non-primary action, include it as a link in the modal's body.
+For short, direct messages the title can include the whole message to add visual clairty to an otherwise repeativite title and body message. When using this style,no other body copy may be included.
```

Comment:
```suggestion
For short, direct messages the title can include the whole message to add visual clarity to an otherwise repetitive title and body message. When using this style, no other body copy may be included.
```